deploy/etc/deployments/zgc_cluster_config.yaml

```diff
@@ -12,5 +12,5 @@ metadata:
   labels:
     name: zgc-cluster-config
 data:
-  cidr: 192.168.100.0/24
+  cidr: 10.0.0.0/24
   MAC_PRIVATE: "0x828300000000"
```

Comment:
This will break current lab setup.
possible make this an parameter/{{}}template for different deploy request? or make KIND network config matches this default

Reply:
Fixed